modules/flowable-http/src/test/java/org/flowable/http/bpmn/HttpServiceTaskTest.java

```diff
@@ -165,30 +158,21 @@ public void testHttpsSelfSigned() {
     @Test
     @Deployment
     public void testConnectTimeout() {
-        try {
-            runtimeService.startProcessInstanceByKey("connectTimeout");
-            fail("FlowableException expected");
-        } catch (final Exception e) {
-            assertTrue(e instanceof FlowableException);
-            assertTrue(e.getCause() instanceof IOException);
-        }
+        assertThatThrownBy(() -> runtimeService.startProcessInstanceByKey("connectTimeout"))
+                .isExactlyInstanceOf(FlowableException.class)
+                .hasCauseInstanceOf(IOException.class);
     }
 
     @Test
     @Deployment
     public void testRequestTimeout() {
-        try {
-            runtimeService.startProcessInstanceByKey("requestTimeout");
-            fail("FlowableException expected");
-        } catch (final Exception e) {
-            assertTrue(e instanceof FlowableException);
-            assertTrue(e.getCause() instanceof SocketTimeoutException
-                    || e.getCause() instanceof SocketException);
-        }
+        assertThatThrownBy(() -> runtimeService.startProcessInstanceByKey("requestTimeout"))
+                .isExactlyInstanceOf(FlowableException.class)
+                .hasCauseInstanceOf(IOException.class);
```

Comment:
The cause here should be `SocketTimeoutException` or `SocketException`

Reply:
I coded it this way because I couldn't figure out how to do an `OR` and both `SocketTImeoutException` and `SocketException` extend from `IOException`.

Reply:
You can do `getCause().isInstanceOfAny(SocketTimeoutException.class, SocketException.class)`.

`getCause()` returns a new `AbstractThrowableAssert` and that has `isInstanceOfAny`.

Reply:
Got it; thanks much.  My one new thing I've learned today :)